CMakeLists.txt

```diff
@@ -235,13 +235,10 @@ endif()
 
 # Add an always out-of-date target to get the Swift compiler version.
 if (SWIFTC_FOUND)
-  set(SWIFT_VERSION ${LLBUILD_OBJ_DIR}/swift-version)
-  add_custom_target(
-      swiftversion ALL
-      COMMAND ${CMAKE_SOURCE_DIR}/utils/write-swift-version ${SWIFTC_EXECUTABLE} ${SWIFT_VERSION}
```

Comment:
We should remove utils/write-swift-version in this change if it is indeed replacing that functionality.